init/arch_specs/eessi_arch_x86.spec

```diff
@@ -1,7 +1,10 @@
 # x86_64 CPU architecture specifications
+# The overview at https://github.com/InstLatx64/InstLatX64_Misc/tree/4802ff02415ed1ecdefddfd540615a27b617349c/SIMD_Euler may be helpful in defining this list
 # Software path in EESSI 	| Vendor ID 	| List of defining CPU features
 "x86_64/intel/haswell"		"GenuineIntel"	"avx2 fma"									# Intel Haswell, Broadwell
-"x86_64/intel/skylake_avx512"	"GenuineIntel"	"avx2 fma avx512f avx512bw avx512cd avx512dq avx512vl"				# Intel Skylake, Cascade Lake
+"x86_64/intel/skylake_avx512"	"GenuineIntel"	"avx2 fma avx512f avx512bw avx512cd avx512dq avx512vl"				# Intel Skylake
+"x86_64/intel/cascadelake"	"GenuineIntel"	"avx2 fma avx512f avx512bw avx512cd avx512dq avx512vl avx512_vnni"	# Intel Cascade Lake
+"x86_64/intel/icelake"	"GenuineIntel"	"avx2 fma avx512f avx512bw avx512cd avx512dq avx512vl avx512_vnni avx512_vbmi2"	# Intel Icelake Lake
```

Comment:
add missing tab/spaces so things are aligned nicely?

Reply:
Fixed in https://github.com/EESSI/software-layer/pull/1084/commits/838f082ca7a02ce12c8c56e5cda642daeca0d786.